code/controllers/subsystem/nightshift.dm

```diff
@@ -58,11 +58,11 @@ SUBSYSTEM_DEF(nightshift)
 			priority_announce("Good morning, crew. As it is now day time, all of the lights aboard the station have been restored to their former brightness.", sound='sound/misc/notice2.ogg', sender_override="Automated Lighting System Announcement")
 		nightshift_active = FALSE
 	var/list/area/affected = return_nightshift_area_types()

```

Comment:
.

Reply:
If that's desired I'll just clean up the whole thing in general, I assume it was left in as some sort of extension hook

Reply:
go ahead.